src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -1,65 +1,60 @@
-import React, { useCallback, useMemo, useState } from "react";
-import { Button, Nav } from "react-bootstrap";
-import { Link } from "react-router-dom";
+import React, { useMemo, useState } from "react";
+import { Nav } from "react-bootstrap";
 import classNames from "classnames";
 
-import AdvancedSettings from "pages/Admin/AdvancedSettings";
-import BasicInfo from "pages/Admin/BasicInfo";
-import EntranceExperience from "pages/Admin/EntranceExperience";
-import VenueDetails from "pages/Admin/Venue/Details";
+import { useVenueId } from "hooks/useVenueId";
+import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
+import { useUser } from "hooks/useUser";
+import { useAdminVenues } from "hooks/useAdminVenues";
 
-import { Venue_v2 } from "types/venues";
+import { LoadingPage } from "components/molecules/LoadingPage";
 
 import "./AdminVenueView.scss";
 
 export enum AdminVenueTab {
-  dashboard = "dashboard",
-  basicInfo = "basic_info",
-  entranceExperience = "entrance_experience",
-  advancedMapSettings = "advanced_map_settings",
+  spaces = "spaces",
+  timing = "timing",
+  run = "run",
 }
 
 const adminVenueTabs: Record<AdminVenueTab, String> = {
-  [AdminVenueTab.basicInfo]: "Start",
-  [AdminVenueTab.entranceExperience]: "Entrance",
-  [AdminVenueTab.advancedMapSettings]: "Advanced",
-  [AdminVenueTab.dashboard]: "Dashboard",
+  [AdminVenueTab.spaces]: "Spaces",
+  [AdminVenueTab.timing]: "Timing",
+  [AdminVenueTab.run]: "Run",
 };
 
-const DEFAULT_TAB = AdminVenueTab.dashboard;
+const DEFAULT_TAB = AdminVenueTab.spaces;
 
-export interface AdminVenueViewProps {
-  venue: Venue_v2;
-}
+export const AdminVenueView: React.FC = () => {
+  const { user } = useUser();
+  useAdminVenues(user?.uid);
```

Comment:
(⚠️ warning, probably a silly question) can I ask why do we need to use this hook here? seems like it fetches all the venues which are current user admin for. But we don't use the list of those venues here in the component, do we?

would like to understand the logic and the reason why it's here.

Reply:
This logic fetches the venues and checks if you are an admin in that venue. Previously you could access directly this page if you had the link for it. Our check was only in `/admin_v2`. If you had the direct link to edit a specific venue EX: `/admin_v2/viktoriasvenue` you could have accessed the page. The requests were probably going to return 403. But it's still leaking information, imo.

Reply:
> This logic fetches the venues and checks if you are an admin in that venue.
Just to be clear, fetching all venues from Firebase and filter after or does the filter happen on the DB level and only get the relevant ones?

Reply:
I’m guessing the hook throws an error/redirects somewhere/etc when forbidden? The name of the hook doesn’t imply that sort of functionality, which makes it a little confusing. I would expect this to return a list of venues I had admin access to or similar based on the name.

Reply:
It's actually `useRoles` and `useIsAdminUser` that's checks for that. I just added them in this commit: https://github.com/sparkletown/sparkle/pull/1511/commits/2afccca403af90d38dca5a81a91041709b2afac3

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1511/commits/d74ce15ca534e3215cda469caf147a7a6d26e9d5